### ISSUE TITLE
wrappped jbang doesn't change the version when you replace the wrapped file

### ISSUE THREAD

**Issue by gustavolira** [jbangdev/jbang#1024]
If you are using a wrapper jbang in your project and replace with a new version, jbang still using the old version.
Even running `./jbang cache clear --all` it doesn'work.
Workaround: `rm -rf ~/.jbang/bin/jbang*`

**Comment by quintesse** [comment]
Have you tried with `jbang wrapper install --force`?

**Comment by maxandersen** [comment]
It seems the jbang script picks up jar from jbang home before looking in wrap location. 

No amount of force install overwrites it.

**Comment by quintesse** [comment]
Indeed, because the wrapper command should only update the local files.
We could perhaps change the priority of the jar lookup: so if there is a local jar we use it before the jar from `~/.jbang/bin`.

**Comment by quintesse** [comment]
In fact, I think that a better option in every case. If somebody installs a wrapper using a certain jbang version we shouldn't decide to use a different one.

**Comment by quintesse** [comment]
Hmmm, it already works that way. In fact I cannot reproduce the problem:

```
$ ../jbang-0.80.2/bin/jbang wrapper install
$ ./jbang version
0.80.2
$ ../jbang-0.80.2.4/bin/jbang wrapper install --force
$ ./jbang version
0.80.2.4
```
